.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules/
 **/yarn-error.log
+.eslintcache
```

Comment:
Maybe we could follow CRA (https://github.com/facebook/create-react-app/pull/9977) and move it into node_modules? It would also be hidden from the IDE's file explorer if `node_modules` is already hidden (which it is in the default `.vscode/settings.json` in the repo)

Reply:
I'm down with that!